interface/modules/zend_modules/module/PrescriptionTemplates/view/layout/layout.phtml

```diff
@@ -0,0 +1,18 @@
+<?php
+use OpenEMR\Core\Header;
+?>
+<!-- Layout -->
+<html>
+    <head>
+        <?php Header::setupHeader(['jquery','bootstrap','datetime-picker','main-theme', 'fontawesome']); ?>
```

Comment:
the only thing you need to call here is datetime-picker. The rest are already brought automatically.

Reply:
Also, do you need to include the code to support the jquery code in ready element to support the datetime-picker?
It looks something like this:
```
        $('.datepicker').datetimepicker({
            <?php $datetimepicker_timepicker = false; ?>
            <?php $datetimepicker_showseconds = false; ?>
            <?php $datetimepicker_formatInput = true; ?>
            <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
            <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
        });
```

Reply:
Actually, not seeing in your code where you even use datetime-picker, so you may be able to remove that from above call.

Reply:
I removed this. datepicker never needed in the print view.